test/fail_compilation/dip25.d

```diff
@@ -2,7 +2,11 @@
 REQUIRED_ARGS: -de
 TEST_OUTPUT:
 ---
-fail_compilation/dip25.d(13): Deprecation: returning `this.buffer[]` escapes a reference to parameter `this`, perhaps annotate with `return`
+fail_compilation/dip25.d(17): Deprecation: returning `this.buffer[]` escapes a reference to parameter `this`
+fail_compilation/dip25.d(17):        perhaps annotate the parameter with `return`
```

Comment:
@Geod24 What parameter? Perhaps we should detect whether the parameter is `this` and in that case say:
```
perhaps annotate the member function with `return`?
```

Reply:
We already do, see the other examples, but apparently the detection is incomplete.

Reply:
Hmm yeah, the ``"perhaps annotate the function with `return`"`` error message looks like it's not triggered